### DISCUSSION TITLE
bug

### DISCUSSION THREAD

**Discussion by Anuragbisenn** [apache/airflow#25505] [General]
### Apache Airflow version

2.3.2

### What happened

i am using [https://airflow.apache.org/api/v1/dags/{dag_id}/dagRuns/{dag_run_id}/taskInstances/{task_id}/{map_index}](https://airflow.apache.org/api/v1/dags/%7Bdag_id%7D/dagRuns/%7Bdag_run_id%7D/taskInstances/%7Btask_id%7D/%7Bmap_index%7D) but i got the error related to {map_index} parameter. it is an integer value like -1, but does not get the response which is expected.
same things happening with other Rest API's which is taking integer as an path peramiter.

### Operating System

linux

### Deployment

Other

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Reply by potiuk** [reply]
See the dicusssion you also (WHY?) opened https://github.com/apache/airflow/discussions/25497. With this level of detail this is at most a  discussion, and until you spend a little more time on describing your issue, there is no way someone will be able to help you to solve your problem.
